src/hearings/models/hearings.enum.ts

```diff
@@ -206,10 +206,15 @@ export enum AnswerSource {
   JUDGE_NAME,
   JUDGE_EXCLUSION,
   HEARING_PANEL,
+  HEARING_PANEL_REQUIRED,
```

Comment:
if this HEARING_PANEL_REQUIRED is created could the HEARING_PANEL being deleleted?